src/OrchardCore.Modules/OrchardCore.Media.Indexing.Pdf/Services/PdfMediaFileTextProvider.cs

```diff
@@ -39,7 +39,7 @@ public async Task<string> GetTextAsync(string path, Stream fileStream)
             {
                 await seekableStream.DisposeAsync();
 
-                await File.DeleteAsync(seekableStream.Name);
+                await Task.Run(() => File.Delete(seekableStream.Name));
```

Comment:
Why did you make this change, while you were requesting the `File.DeleteAsync()`

Reply:
DeleteAsync does not exists.